js_modules/dagit/packages/core/src/ui/Filters/FilterDropdown.tsx

```diff
@@ -118,10 +201,10 @@ export const FilterDropdown = ({filters, setIsOpen, setPortaledElements}: Filter
           type="text"
           value={search}
           onChange={(e) => setSearch(e.target.value)}
+          onKeyUp={handleKeyUp}
```

Comment:
Keydown instead of keyup, so that I could potentially hold the key down to go through a long list of options?